Packs/Campaign/ReleaseNotes/3_2_23.md

```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+
+##### Detect & Manage Phishing Campaigns
+
+Changed the playbook to use the *Email HTML* field instead of the *Email Body HTML* by default. We recommend that users change to the value locally as well. This change should not affect the functionality of the playbook.
```

Comment:
```suggestion
Changed the playbook to use the *Email HTML* field instead of the *Email Body HTML* by default. We recommend that users change the value locally as well. This change should not affect the functionality of the playbook.
```